common/config/config.go

```diff
@@ -37,6 +37,7 @@ import (
 	"github.com/uber/cadence/common/dynamicconfig/dynamicproperties"
 	"github.com/uber/cadence/common/peerprovider/ringpopprovider"
 	"github.com/uber/cadence/common/service"
+	"github.com/uber/cadence/service/sharddistributor/config"
```

Comment:
Common folder should not depend on service specific folders. One way to prevent this is to define shard distributor config in this file and implement an adaptor to convert it to the type in sharddistributor package.

Reply:
Ack. Will remove it for now. And will add a copy-pasted version and handle it in the cmd/server in a separate PR.